cmd/heimdallcli/stake.go

```diff
@@ -48,11 +55,26 @@ func StakeCmd() *cobra.Command {
 				return err
 			}
 
+			route := fmt.Sprintf("custom/%s", checkpointEndpoint)
```

Comment:
it will be as `custom//chainmanager/params`. Remove extra `/` from `checkpointEndpoint`